Documentation/bpf.rst

```diff
@@ -3220,16 +3326,22 @@ the following lists native and offloaded XDP drivers as of kernel 4.17.
 
   * qede
 
-.. [1] XDP available via out of tree driver as of kernel 4.17, but will be
-   upstreamed soon.
-
 **Drivers supporting offloaded XDP**
 
 * **Netronome**
 
-  * nfp
+  * nfp [2]_
+
+LLVM and iproute2 examples for writing and loading XDP programs are included
+in the toolchain section. Further code samples for getting started can also
+be found in the Linux kernel's BPF selftest suite:
```

Comment:
I can't really tell from the PR but this seems like it might make sense somewhere else in the docs. If it does, we should move it.